docs/IE_DG/Integrate_with_customer_application_new_API.md

```diff
@@ -217,7 +217,7 @@ add_executable(${PROJECT_NAME} src/main.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE ${InferenceEngine_LIBRARIES} ${OpenCV_LIBS} ${NGRAPH_LIBRARIES})
 ```
 3. **To build your project** using CMake with the default build tools currently available on your machine, execute the following commands:
-> **NOTE**: Make sure **Set the Environment Variables** step in [OpenVINO Installation](../../inference-engine/samples/hello_nv12_input_classification/README.md) document is applied to your terminal, otherwise `InferenceEngine_DIR` and `OpenCV_DIR` variables won't be configured properly to pass `find_package` calls.
+> **NOTE**: Make sure **Set the Environment Variables** step in [OpenVINO Installation](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode#installing) document is applied to your terminal, otherwise `InferenceEngine_DIR` and `OpenCV_DIR` variables won't be configured properly to pass `find_package` calls.
```

Comment:
I suppose this link is also incorrect. We need something like https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html#set-the-environment-variables